src/network/bmproto.py

```diff
@@ -201,51 +212,59 @@ def decode_simple(self, char="v"):
         # retval (array)
         parserStack = [[1, 1, False, pattern, 0, []]]
 
-        #try:
+        # try:
```

Comment:
This section can be removed, it's from when I was debugging for the "maximum stack exceeded" exception, in the end it was in the garbage collector so it's completely unrelated)